crates/mr_bundle/src/bundle.rs

```diff
@@ -18,7 +18,7 @@ pub type ResourceMap = HashMap<PathBuf, ResourceBytes>;
 /// by the receiver.
 ///
 /// The manifest may describe locations of resources not included in the Bundle.
-#[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
```

Comment:
I designed this type specifically to be `!Clone`, because as a general purpose tool, the bundles may be huge. If you need a cloneable wrapper around this, the bundle should go in the appropriate wrapper type: `Rc`, `Arc`, `Arc<Mutex<_>>`, etc.

Does that work in your case?

```suggestion
#[derive(Debug, PartialEq, Eq, Serialize, Deserialize)]
```